js/src/env.ts

```diff
@@ -0,0 +1,16 @@
+import { getEnvironmentVariable } from "./utils/env.js";
+
+export const isTracingEnabled = (tracingEnabled?: boolean): boolean => {
+  if (tracingEnabled !== undefined) {
+    return tracingEnabled;
+  }
+  const envVars = [
+    "LANGSMITH_TRACING_V2",
+    "LANGCHAIN_TRACING_V2",
+    "LANGSMITH_TRACING",
+    "LANGCHAIN_TRACING",
+  ];
+  return Boolean(
```

Comment:
nit: I'm not sure what this function does, I usually prefer `!!`

Reply:
Should be the same, replaced